kafka-questdb-connector-samples/stocks/docker-compose.yml

```diff
@@ -85,4 +84,4 @@ services:
       BOOTSTRAP_SERVERS: "kafka:9092"
       CONFIG_STORAGE_TOPIC: "debezium_connect_config"
       OFFSET_STORAGE_TOPIC: "debezium_connect_offsets"
-      STATUS_STORAGE_TOPIC: "debezium_connect_status"
\ No newline at end of file
+      STATUS_STORAGE_TOPIC: "debezium_connect_status"
```

Comment:
@jerrinot I am not sure why this happened...

Reply:
this is actually a good thing. 

there is an ancient custom that source code/config files should end with an empty line. 
this wasn't the case here and this PR is fixing it. I guess your formatting tool did that automatically.